common/src/main/java/com/wynntils/gui/screens/overlays/OverlayManagementScreen.java

```diff
@@ -71,9 +71,8 @@ public class OverlayManagementScreen extends Screen {
     private static final Component CLOSE_TOOLTIP_LINES =
```

Comment:
The name of these constants made sense when they were a List<Component>, but now I think they should be just CLOSE_TOOLTIP etc.

Reply:
Done